R/zzz.R

```diff
@@ -0,0 +1,4 @@
+.onLoad <- function(libname, pkgname) {
```

Comment:
Why is this file named so weird? Is this a leftover from development and debuging?

Reply:
Standarrd naming convetion for [functions that load external files](https://r-pkgs.org/code.html#sec-code-r-landscape)